Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG PARENT_VERSION=2.3.0-node20.15.0
-ARG PORT=3000
+ARG PORT=3023
```

Comment:
Following our previous discussion on not using the same port number for ffc-sfd-xxx and the corresponding fcp-fd-xxx, I think we can follow a pattern of adding 20 to existing ffc-sfd-xxx ports to derive corresponding fcp-fd-xxx. If we go by this pattern, there will be conflict between  fcp-fd-comms and  fcp-fd-business with the port number that you have used..

Reply:
Update has been made to `3039` and `9039`.